docs/about/whats-new.ipynb

```diff
@@ -662,6 +796,14 @@
     "- Vector or matrix elements can now be accessed and modified directly using the new `fields` property of variables.\n",
     "  `fields` provides access to vector elements `x`, `y`, and `z` or matrix elements `xx`, `xy`, ..., `zz`.\n",
     "    \n",
+    "</div>\n",
+    "\n",
+    "<div class=\"alert alert-info\">\n",
+    "\n",
+    "**New in 0.8**\n",
+    "\n",
+    "The `fields` property can now be iterated and behaves similar to a (read-only) `dict`.\n",
```

Comment:
Why read-only? Values can be set. Only the set of keys is immutable.

Reply:
Because you cannot add items to the dict, i.e., the dict is read-only and has no `__setitem__`.

Reply:
It does:
https://github.com/scipp/scipp/blob/4ecff545e80c2bfc5cae7cb9d5a8569c291551c9/python/src/scipp/_structured.py#L39-L40

Reply:
It gives an exception, you cannot add new fields, try:
```
import scipp as sc
vec = sc.vector(value=[1,2,3])
vec.fields['abc']
```

Reply:
Do you have a suggestion how to formulate this better, so it becomes clearer? Just remove the bit in parentheses?

Reply:
I think in this place it is ok to just say 'similar to a dict'. Or if you want to be more precise, "The `fields` property can now be iterated and behaves similar to a `dict` with fixed keys."